assets/airports/egnm.json

```diff
@@ -10,251 +10,2015 @@
+            ]
+        }
+    ],
     "wind": {
         "angle": 270,
         "speed": 10
     },
     "arrivalRunway": "32",
     "departureRunway": "32",
     "fixes": {
-        "_EGNM165050": ["N53.43199862756", "W001.46564026324"],
-        "_EGNM300050": ["N54.08842200093", "W002.32289085237"],
-        "_EGNM325050": ["N54.23291311084", "W002.10125502998"],
-        "_EGNM360050": ["N54.31523430050", "W001.66076939999"],
-        "EGNM" : ["N53d51m57.69",  "W001d39m38.77"],
-        "D082N": ["N53d47m02.00",  "W001d42m94.00"],
-        "D082J": ["N53d46m34.00",  "W001d49m59.00"],
-        "POL"  : ["N53d44m38.00",  "W002d06m12.00"],
-        "NELSA": ["N53d51m48.00",  "W002d11m05.00"],
-        "05DME": ["N53d52m66.00",  "W001d40m71.00"],
-        "21DME": ["N53d53m67.00",  "W001d42m97.00"],
-        "35DME": ["N53d54m08.00",  "W001d45m35.00"],
-        "D052L": ["N53d52m31.00",  "W001d50m65.00"],
-        "D052K": ["N53d51m67.00",  "W001d51m95.00"],
-        "D272J": ["N53d51m86.00",  "W001d54m64.00"],
-        "R255" : ["N53d50m20.00",  "W001d48m24.00"],
-        "GAM38": ["N53d44m88.00",  "W001d39m79.00"],
-        "GAM35": ["N53d42m68.00",  "W001d36m37.00"],
-        "GAM31": ["N53d39m74.00",  "W001d31m81.00"],
-        "GAM19": ["N53d30m91.00",  "W001d18m20.00"],
-        "GAM18": ["N53d30m18.00",  "W001d17m07.00"],
-        "GAM15": ["N53d28m74.00",  "W001d12m16.00"],
-        "DOPEK": ["N53d23m04.00",  "W000d55m32.00"],
-        "LAMIX": ["N53d18m48.00",  "W000d40m93.00"],
-        "D140B": ["N53d50m14.00",  "W001d36m85.00"],
-        "GAM36": ["N53d46m54.00",  "W001d30m90.00"],
-        "GAM32": ["N53d43m26.00",  "W001d27m08.00"],
-        "GAM29": ["N53d40m79.00",  "W001d24m22.00"],
-        "GA19A": ["N53d32m56.00",  "W001d14m72.00"],
-        "GA15A": ["N53d29m26.00",  "W001d10m93.00"],
-        "GAM14": ["N53d28m44.00",  "W001d09m99.00"],
-        "LBA"  : ["N53d51m54.00",  "W001d39m10.00"]
+        "_GAM15DME": ["N53d28m25", "W001d12m42"],
+        "_GAM319018": [53.50334238484, -1.28370756576],
+        "_GAM319019": [53.51564501839, -1.30250407183],
+        "_GAM319031": [53.66304347462, -1.52891347974],
+        "_GAM319035": [53.71208001909, -1.60473424361],
+        "_GAM319038": [53.74882567603, -1.66171547265],
+        "_GAM327014": [53.47423039856, -1.16582663874],
+        "_GAM327015": [53.48798922439, -1.18151693089],
+        "_GAM327019": [53.54300356034, -1.24437981498],
+        "_GAM327029": [53.68039210836, -1.40225184649],
+        "_GAM327032": [53.72156739109, -1.44981368631],
+        "_GAM327036": [53.77643789435, -1.51337415705],
+        "_GAM327POL118": [53.45102, -1.13938],
+        "_GAMR319INTPOLR118": [53.48876, -1.26144],
+        "_ILBF2DME": [53.83349605283, -1.61072235541],
+        "_ILF0.5DME": [53.88059032404, -1.68318520718],
+        "_ILF2.1DME": ["N53d53m40", "W001d42m58"],
+        "_ILF3.5DME": ["N53d54m05", "W001d45m21"],
+        "_INTGAM319": [53.83812565585, -1.80078584616],
```

Comment:
These intercept fixes should be defined as a bearing and distance from a fix, i.e., `_GAM319040` instead of `_INTGAM319`

Reply:
Not sure of the distances, but I'll try to find them

Reply:
The coordinates you have work fine, the name of the fixes is the only thing that needs to be changed. You can just calculate the distances between the VOR and the fix using GPSVisualizer.